autoload/neomake/makers/ft/racket.vim

```diff
@@ -0,0 +1,17 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#racket#EnabledMakers()
+    return ['raco']
+endfunction
+
+" This is the same form of syntax-checking used by DrRacket as well. The
+" downside is that it will only catch the first error, but none of the
+" subsequent ones. This is due to how evaluation in Racket works.
+function! neomake#makers#ft#racket#raco()
+    return {
+        \ 'exe': 'raco',
+        \ 'args': ['expand', '%:p'],
```

Comment:
`append_file` (default) will append the current file already, so the `%:p` should be removed?!